### PR TITLE
Fixed #321 Hitting enter in the password input saves the user but not the password

### LINKED ISSUE #321: Hitting enter in the password input saves the user but not the password
In the edit user screen if you enter a new password for a user it will save the user but not the password. You need to hit "Save" in order to save the password (and then get the confirmation popup).

This should probably either behave the same in both cases or enter shouldn't cause the submit in the password field.

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Fixes an issue where hitting the enter button wouldn't save a user and instead re-directs you to the users list-view.

Issue reference: #321 

## How To Test This

1. Admin -> user management -> new user.
2. Fill out the username, last name, email and password fields.
3. Hit enter.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by jack7anderson7** [comment]
Assessed :+1:

**Comment by cameronblaikie** [comment]
Assessed :+1:

File: modules/Users/views/view.edit.php [hunk 1]
```diff
@@ -219,6 +219,22 @@ public function display()
                     <input type="button" id="SAVE_HEADER" title="{$APP['LBL_SAVE_BUTTON_TITLE']}" accessKey="{$APP['LBL_SAVE_BUTTON_KEY']}"
                           class="button primary" onclick="var _form = $('#EditView')[0]; if (!set_password(_form,newrules('{$minpwdlength}','{$maxpwdlength}','{$REGEX}'))) return false; if (!Admin_check()) return false; _form.action.value='Save'; {$CHOOSER_SCRIPT} {$REASSIGN_JS} if(verify_data(EditView)) _form.submit();"
                           name="button" value="{$APP['LBL_SAVE_BUTTON_LABEL']}">
+EOD
```
Comment (samus-aran) [inline modules/Users/views/view.edit.php]: Why are we not amending the original $action_button_header? It pretty much looks identical, what is the difference?

Comment (Dillon-Brown) [reply modules/Users/views/view.edit.php]: Each $action_button_header[] element is for a different button (Save, cancel, etc). This just adds a script to the page to handle submitting through the keyboard without clicking a button.

**Pull request merged**